src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html

```diff
@@ -162,5 +162,9 @@
 <H2>X-ChromeLogger-Data Header Information Leak</H2>
 This checks response headers for the presence of X-ChromeLogger-Data or X-ChromePhp-Data details.
 
+<H2>X-Debug-Token Information Leak</H2>
+This checks response headers for the presence of X-Debug-Token and X-Debug-Token-Link details.
+Which suggest the use/exposure of Symfony's Profiler, which can leak a significant amount of information of interest to Security Testers and malicious individuals.
```

Comment:
Good point. For the first one "Which indicates..." ? For the second one I'll remove "can", there's no reason to be non-commital, it really does happen.